public/js/itemList.js

```diff
@@ -171,6 +171,11 @@ function getCharacterList(data, type, itemId) {
                         data-timestamp="${ character.pivot.created_at }"
                         data-is-short="1">
                     </span>
+                    ${character.is_alt ? `
+                        <span class="font-weight-bold">
+                            <span class="role-circle" style="color:orange">Alt</span>
+                        </span>
+                    ` : ''}

```

Comment:
Please change to:
```
<span class="font-weight-bold">
    <span class="text-legendary">Alt</span>
</span>
```

And move between lines 169 and 170.

![image](https://user-images.githubusercontent.com/3407841/90581828-13f3d880-e181-11ea-825f-165a83225786.png)

![image](https://user-images.githubusercontent.com/3407841/90581836-18b88c80-e181-11ea-8e5f-3f686e4d1028.png)